models/event.go

```diff
@@ -21,13 +21,13 @@ import (
 
 // Event represents a single measurable event read from a device
 type Event struct {
-	ID          string    `json:"id" codec:"id,omitempty"`             // ID uniquely identifies an event, for example a UUID
-	Pushed      int64     `json:"pushed" codec:"pushed,omitempty"`     // Pushed is a timestamp indicating when the event was exported. If unexported, the value is zero.
-	Device      string    `json:"device" codec:"device,omitempty"`     // Device identifies the source of the event, can be a device name or id. Usually the device name.
-	Created     int64     `json:"created" codec:"created,omitempty"`   // Created is a timestamp indicating when the event was created.
-	Modified    int64     `json:"modified" codec:"modified,omitempty"` // Modified is a timestamp indicating when the event was last modified.
-	Origin      int64     `json:"origin" codec:"origin,omitempty"`     // Origin is a timestamp that can communicate the time of the original reading, prior to event creation
-	Readings    []Reading `json:"readings" codec:"readings,omitempty"` // Readings will contain zero to many entries for the associated readings of a given event.
+	ID          string    `json:"id,omitempty" codec:"id,omitempty"`             // ID uniquely identifies an event, for example a UUID
```

Comment:
Should you be adding omitempty to codec tag?

Calling out once -- occurs in multiple places.

Reply:
No. Copy paste error. Good catch, thanks.